bower.json

```diff
@@ -13,6 +13,7 @@
     "normalize-scss": "^3.0.3",
     "flickity": "^2.0.5",
     "babel-polyfill": "^0.0.1",
+    "pdfjs-embed": "eScholarship/pdfjs-embed",
```

Comment:
Is there a way we can specify the particular changeset id to bower (like we do in package.json) to force it to pick up a new version when we change CSS or image resources in pdfjs-embed?